### DISCUSSION TITLE
Exploring Pixi Asset Pack 1.0 – Benefits and Current Limitations

### DISCUSSION THREAD

**Discussion by furic** [prefix-dev/pixi#2012] [Ideas]
It's exciting to see that Pixi's Asset Pack has officially reached [version 1.0](https://pixijs.com/blog/assetpack-1.0.0). I've taken a deep dive into it to assess whether it can benefit our projects. So far, it looks promising, though there are still a few issues that need to be addressed. I'd love to discuss it here and hear if anyone else has had similar experiences.

Reasons to Use Asset Pack:

- [Texture Sprite Packing](https://pixijs.io/assetpack/docs/guide/pipes/texture-packer/): No more manual work with Texture Packer.
- [Automated Compression & File Formats](https://pixijs.io/assetpack/docs/guide/pipes/compress/): Keeps a 100% PNG, while automatically generating compressed PNG and WebP versions.
- [Mipmaps Generation](https://pixijs.io/assetpack/docs/guide/pipes/mipmap/): Devices with smaller resolutions can use lower-res textures for better performance.
- [Bundle Manifest Generation](https://pixijs.io/assetpack/docs/guide/pipes/manifest/): No more hard-coded asset lists.
- [Hash Versioning](https://pixijs.io/assetpack/docs/guide/pipes/cache-buster/): Ensures the latest files are always used.
- Artist-Friendly Workflow: Artists only manage raw assets, while export automation handles the rest.

Issues/Limitations:

- ~~Spine Atlas Export Issues: Spine atlas not correctly export (submitted a [PR](https://github.com/EsotericSoftware/spine-runtimes/pull/2619) for spine-runtime).~~
- Bitmap Font Export Problems: Bitmap fonts are not exporting properly (submitted an [issue](https://github.com/pixijs/assetpack/issues/88)).
- Sound Sprite Support: Currently unsupported (submitted an [issue](https://github.com/pixijs/assetpack/issues/83)).

While Asset Pack is still in its early stages, I believe it has great potential to become an essential tool for Pixi development. Has anyone else worked with it? I'd love to hear your thoughts and experiences!

**Comment by wolfv** [comment]
Hi @furic - sorry but I think you unfortunately reached the wrong repository :) You are probably talking about pixi.js? 

Cheers!
